kumascript/macros/js_property_attributes.ejs

```diff
@@ -9,104 +9,103 @@
 // 
 // {{js_property_attributes(1,1,0)}}
 
-var header;
-var writable_name;
-var enumerable_name;
-var configurable_name;
-var yes;
-var no;
+const text = mdn.localStringMap({
+  "en-US": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
+  },
+  "es": {
+    header: `Atributos de la propiedad <code>${env.title}</code>`,
+    writableName: "Sobrescribir",
+    enumerableName: "Numerable",
+    configurableName: "Configurable",
+    yes: "Sí",
+    no: "No"
+  },
+  "fr": {
+    header: `Attributs de <code>${env.title}</code>`,
+    writableName: "Écrivable",
+    enumerableName: "Énumérable",
+    configurableName: "Configurable",
+    yes: "Oui",
+    no: "Non"
+  },
+  "ja": {
+    header: `<code>${env.title}</code> のプロパティ属性`,
+    writableName: "書込可能",
+    enumerableName: "列挙可能",
+    configurableName: "設定可能",
+    yes: "可",
+    no: "不可"
+  },
+  "ko": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
+  },
+  "pt-BR": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
+  },
+  "ru": {
+    header: `Атрибуты свойства <code>${env.title}</code>`,
+    writableName: "Записываемое",
+    enumerableName: "Перечисляемое",
+    configurableName: "Настраиваемое",
+    yes: "да",
+    no: "нет"
+  },
+  "zh-CN": {
+    header: `<code>${env.title}</code> 的属性特性`,
+    writableName: "可写",
+    enumerableName: "可枚举",
+    configurableName: "可配置",
+    yes: "是",
+    no: "否"
+  },
+  "zh-TW": {
+    header: `<code>${env.title}</code> 的屬性特性`,
+    writableName: "可寫",
+    enumerableName: "可列舉",
+    configurableName: "可配置",
+    yes: "是",
+    no: "否"
```

Comment:
poke @irvin for a review

Reply:
@irvin Feel free to open a follow-up PR. 🙏